pkg/apis/build/v1alpha1/build_pod.go

```diff
@@ -209,7 +209,7 @@ func (b *Build) BuildPod(config BuildPodImages, secrets []corev1.Secret, buildPo
 				{
 					Name:    "detect",
 					Image:   builderImage,
-					Command: []string{"/lifecycle/detector"},
+					Command: []string{"/cnb/lifecycle/detector"},
```

Comment:
We should probably keep executing `/lifecycle` directly and follow the lead of pack and the [tekton catalog](https://github.com/tektoncd/catalog/blob/master/buildpacks/buildpacks-v3.yaml) if/when we move to /cnb/lifecycle

Reply:
This will require us to add a symlink but, it will allow our builders to still be used with pack.